### PR TITLE
[PT/XLA] Fix tags of GPU images

### PR DESCRIPTION
**Pull request by will-cromar**
# Description

Update image tags for PyTorch/XLA GPU images.

- Fix prefix for 2.0 images (`2.0` -> `r2.0`)
- Fix CUDA version for nightly tests

# Tests

- http://shortn/_zKjdegaiGD

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [x] I have performed a self-review of my code.
- [x] I have necessary comments in my code, particularly in hard-to-understand areas.
- [x] I have run one-shot tests and provided workload links above if applicable. 
- [x] I have made or will make corresponding changes to the doc if needed.

### CODE REVIEW

**Review by steventk-g — APPROVED** [review]

File: tests/pytorch/nightly/common.libsonnet [hunk 1]
```diff
@@ -117,7 +117,7 @@ local volumes = import 'templates/volumes.libsonnet';
   },
   GpuMixin:: {
     local config = self,
-    imageTag+: '_cuda_11.2',
+    imageTag+: '_cuda_11.7',
```
Comment (steventk-g) [inline tests/pytorch/nightly/common.libsonnet:120]: NIT: Why 11.7 and not 11.8?

Comment (will-cromar) [reply tests/pytorch/nightly/common.libsonnet:120]: We have builds for both. Is 11.8 working now?

Comment (steventk-g) [reply tests/pytorch/nightly/common.libsonnet:120]: Not sure, that's why I was curious 😁

Comment (will-cromar) [reply tests/pytorch/nightly/common.libsonnet]: Let's try 11.8. The test is broken anyway :smile:

**Pull request merged**